l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -109,3 +113,35 @@ def open(self, mode: str = "r", cached: bool = True, cache_size_bytes: int = int
         self.frames = self.root[FRAME_ARRAY_KEY]
         self.agents = self.root[AGENT_ARRAY_KEY]
         self.scenes = self.root[SCENE_ARRAY_KEY]
+
+    def __repr__(self) -> str:
+        fields = [
+            "Num Scenes",
+            "Num Frames",
+            "Num Agents",
+            "Total Time (hr)",
+            "Avg Frames per Scene",
+            "Avg Agents per Frame",
+            "Avg Scene Time (sec)",
+            "Avg Frame frequency",
+        ]
+        if len(self.agents) > 1:
+            times = self.frames[1:10]["timestamp"] - self.frames[0:9]["timestamp"]
+            frequency = np.mean(1 / (times / 1e9))  # from nano to sec
+        else:
+            frequency = 0  # can't read the frequency
+
+        values = [
+            len(self.scenes),
+            len(self.frames),
+            len(self.agents),
+            len(self.frames) / max(frequency, 1e-9) / 3600,
+            len(self.frames) / max(len(self.scenes), 1e-9),
+            len(self.agents) / max(len(self.frames), 1e-9),
+            len(self.frames) / max(len(self.scenes), 1e-9) / 10,
```

Comment:
Doesn't 10 stem from our sampling frequency and thus needs be changed as well?

Reply:
good catch :)

Reply:
In case `len(self.scenes) = 0` then `len(self.frames)` should also be 0.
To avoid divided by zero, I would simply put 1 in the denominator:
            len(self.frames) / max(len(self.scenes), 1),
            len(self.agents) / max(len(self.frames), 1),
            len(self.frames) / max(len(self.scenes), 1) / frequency,

Reply:
For well designed zarr that's true, so I think we should consider that to be always true. 1e-9 or 1 doesn't really make any difference if the numerator is 0, but let's put 1